keras/optimizers.py

```diff
@@ -637,21 +637,31 @@ def from_config(self, config):
 
 if K.backend() == 'mxnet':
     class SGD(Optimizer, K.mx.optimizer.SGD):
-        def __init__(self, lr=0.01, momentum=0., **kwargs):
+        def __init__(self, lr=0.01, momentum=0., clipnorm=None, clipvalue=None, nesterov=False, **kwargs):
```

Comment:
why you add more named parameter here? It can always passed via kwargs. It is better that we keep this interface same as original one.